### ISSUE TITLE
Expedition Damage Widget only for my damage?

### ISSUE THREAD

**Issue by zr0fve** [HunterPie/HunterPie#505]
I have installed Stracker's Loader and it seems to work if I'm looking at the logs correctly. However the guiding lands/expedition damage widget still seems to be half functional. I only see my own DPS the others stay at 0.0 damage

[ 07:03:28 ] Found config file
[ 07:03:28 ] Loading plugin "nativePC\\plugins\\!CRCBypass.dll"
[ 07:03:28 ] CRC Bypass Loading...
[ 07:03:28 ] Patch of CRC Bypass Loaded!
[ 07:03:28 ] DONE !
[ 07:03:28 ] Loading plugin "nativePC\\plugins\\MonsterLoader.dll"
[ 07:03:28 ] Loading plugin "nativePC\\plugins\\QuestLoader.dll"
[ 07:03:28 ] Hooking Quest Loader

**Comment by Haato3o** [comment]
The damage meter works by reading the damage you've done to the quest objectives. Since GL/Expeditions are not quests and have no objectives, the game doesn't track your damage in those, so HunterPie uses its fallback that hooks the damage function and starts calculating the damage from that.

The damage function only processes the local player damage, therefore you'll only be able to see your own damage.

TL;DR: Game limitation, nothing can be done about that.
